app/models/concerns/hud_reports/util.rb

```diff
@@ -21,6 +21,15 @@ module HudReports::Util
       end.map(&:enrollment_group_id).uniq
     end
 
+    def pit_date(month:, before:)
+      year = before.year if month < before.month
```

Comment:
If the selected month is earlier in the year than the date we're trying to occur before, use the year from the date we want to be before.
If the date occurs in the month we're trying to occur before and the day we want to occur before is after the last Wednesday of the month, use the year from the date we want to be before.
if the month is later in the year than the date we're trying to be before, use the prior year
if the month is the same as the date we're trying to be before, and the date falls before the third Wednesday, use the previous year.